en/docs/deploy/configure-admin-banner.md

```diff
@@ -0,0 +1,29 @@
+# Admin Advisory Banner
+
+WSO2 IS can be configured to display an advisory banner for admin login pages. Once enabled, a banner will be displayed in the Management Console login and Console Application login pages with the configured administrative message. The super tenant banner will be applicable for all tenants.
```

Comment:
```suggestion
WSO2 Identity Server can be configured to display an advisory banner for admin login pages. Once enabled, a banner will be displayed on the login page of the Management Console login with the configured administrative message. The super tenant banner will be applicable to all tenants.
```

Reply:
Removed the mentioning of the **console application** as we have not publicly documented it in our docs.